File: python/alphabet.py
def listAlphabet(pangram):
    listAlphabet =  list(map(chr, range(97, 123)))
    pangramLowerCase = pangram.lower()
    for letter in listAlphabet:
        if letter not in pangramLowerCase:
            print('False')

    print('True')

pangram = "Cwm fjord bank glyphs vext quiz"
listAlphabet(pangram)

File: js/alphabet.js
function Alphabet(pangram) {
  let alphabet = String.fromCharCode(...Array(123).keys())
    .slice(97)
    .split(",");
  pangramLowerCase = pangram.toLowerCase;
  for (const letter of alphabet) {
    const result = alphabet.filter((letter) => letter === pangramLowerCase);
  }

  console.log(alphabet);
}

pangram = "Cwm fjord bank glyphs vext quiz";
Alphabet(pangram);
//
